internal/pkg/remote/endpoint/client_test.go

```diff
@@ -112,7 +110,7 @@ func TestKeyserverClientOpts(t *testing.T) {
 		{
 			name: "Fake cloud",
 			endpoint: &Config{
-				URI: "cloud.nonexistent-xxxx-domain.io",
+				URI: "cloud.inexistent-xxxx-domain.io",
```

Comment:
@DrDaveD @ikaneshiro this is misspelled and also not the canonical IETF recommended dummy domain of `example.com`

Reply:
Replaced with cloud.example.com